common/test/unit/com/thoughtworks/go/domain/AgentConfigTest.java

```diff
@@ -55,9 +57,9 @@ public void shouldValidateIpV4AndIpV6() throws Exception {
 
     @Test
     public void shouldDetectInvalidIPAddress() throws Exception {
-        shouldBeInvalid("blahinvalid", "IpAddress is invalid.");
-        shouldBeInvalid("blah.invalid", "IpAddress is invalid.");
-        shouldBeInvalid("399.0.0.1", "IpAddress is invalid.");
+        shouldBeInvalid("blahinvalid", "'blahinvalid' is an invalid IP address.");
+        shouldBeInvalid("blah.invalid", "'blah.invalid' is an invalid IP address.");
+        shouldBeInvalid("399.0.0.1", "'399.0.0.1' is an invalid IP address.");
```

Comment:
Get rid of or replace all `shouldBeInvalid` and `shouldBeValid` tests with the new test?